python/cudf/cudf/core/dataframe.py

```diff
@@ -6998,7 +6998,7 @@ def explode(self, column, ignore_index=False):
         return super()._explode(column, ignore_index)
 
     def pct_change(
-        self, periods=1, fill_method="ffill", limit=None, freq=None
+        self, periods=1, fill_method=no_default, limit=no_default, freq=None
```

Comment:
Just a heads up, in pandas we might allow `fill_method=None` but discussion is underway https://github.com/pandas-dev/pandas/issues/53491#issuecomment-1727798279

Reply:
Sounds good, let's do the modification in this branch once pandas ships this change?